drainer/pump.go

```diff
@@ -176,6 +177,7 @@ func (p *Pump) PullBinlog(pctx context.Context, last int64) chan MergeItem {
 				p.reportErr(pctx, err)
 				return
 			}
+			resp.Entity.Payload = resp.Entity.Payload[:0]
```

Comment:
what's the purpose of it

Reply:
to make sure this array can be GC asap

Reply:
maybe resp.Entity.Payload = nil . I'm afraid a zero length slice will still hold the reference to underlying array